src/emulator/types.ts

```diff
@@ -32,9 +33,10 @@ export const ALL_SERVICE_EMULATORS = [
   Emulators.DATABASE,
   Emulators.HOSTING,
   Emulators.PUBSUB,
+  Emulators.LOGGING,
 ];
 
-export const EMULATORS_SUPPORTED_BY_GUI = [Emulators.DATABASE];
+export const EMULATORS_SUPPORTED_BY_GUI = [Emulators.DATABASE, Emulators.LOGGING];
```

Comment:
This seems maybe backwards?  Logging emulator is (in some ways) part of the GUI not something that should cause the GUI to run.

Also this is not your fault but this array is out of date.  Should include `FIRESTORE` and `FUNCTIONS` right?